package.json

```diff
@@ -28,7 +28,7 @@
     "level": "~5.0.1",
     "level-sublevel": "~6.6.5",
     "node-forge": "^0.7.1",
-    "simple-glob": "^0.1",
+    "simple-glob": "^0.2.0",
```

Comment:
Are these package updates needed for the tests? Cos they should be in a separate PR

Reply:
They shouldn't be needed (unless sinon has dependency for their specific version which I don't think is the case), I'll try only leaving the new package and revert `simple-glob` and `eslint-plugin-react` to their original versions.